### PR TITLE
Code Kingdoms Editor Accessibility

### PR DESCRIPTION
**Pull request by inversion**
This is not intended to be merged yet, using for discussion in https://github.com/bbc/microbit-extras/issues/3049 and related issues.

### CODE REVIEW

File: editor/external.ts [hunk 1]
```diff
@@ -750,6 +753,7 @@ module TDev {
       // rename variable prompt
       iframe.setAttribute("sandbox", "allow-modals allow-scripts allow-same-origin allow-popups");
       iframe.addEventListener("load", () => {
+
```
Comment (inversion) [inline editor/external.ts]: Initially tried adding the `iframe.focus()` call here but the focus was stolen back to `body`. I couldn't figure out where that was occurring easily - I investigated the explicit focus-changing code in TouchDevelop but couldn't see anything relevant. Perhaps it is when the simulator finishes loading?

File: editor/external.ts [hunk 2]
```diff
@@ -366,6 +367,8 @@ module TDev {
       }
 
       public receive(event) {
+
+        if (!this.iframeFocused) { this.iframe.focus(); this.iframeFocused = true; }
```
Comment (inversion) [inline editor/external.ts]: Bit of a hack to make the external editor iframe focus every time it sends an event to TD. See comment below (open to suggestions).

**Comment by pelikhan** [comment]
@inversion, could you rebase this PR to the ``bbcmicrobitmu`` branch which is currently used in live. I will propagate those changes in all the branches.

**Comment by inversion** [comment]
@pelikhan Yes, now done.

**Comment by pelikhan** [comment]
Still seems to be pointing at master. Could you fix the conflicts?

**Comment by inversion** [comment]
Yep, forgot to make the new PR - https://github.com/Microsoft/TouchDevelop/pull/478

**Pull request closed without merging**